ext/pdo_pgsql/tests/bug_33876.phpt

```diff
@@ -148,5 +151,5 @@ Array
 (
     [0] => 22P02
     [1] => 7
-    [2] => %s:  %sboolean%s
+    [2] => ERROR:  invalid input syntax for type boolean: ""
```

Comment:
This test has recently [been fixed wrt. l10n issues](ac8e5e12b8c3476e950ff0c812110ee3ddefef74). Did that not work for you?

Reply:
The link doesn't work. 

When I replaced all of the files this apparently was one of the files that got changed when replacing. Now when I try to test this, the test gets skipped because of a connection error. So I can undo this change if you believe it was correct already.

Reply:
Link fixed. :) And yes, please undo this change. Your variant will work for an English server (and as long they don't change the exact error message), but not for others (I hit this back then with a German server).

Reply:
Ok. Undid the change for this test and for the other 2 tests (`copy_from.phpt` and `copy_to.phpt`).